### PR TITLE
create env store

### PR DESCRIPTION
**Pull request by ubozov**
Resolves BV-1083.

### CODE REVIEW

File: env/env.go [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+import (
+	"encoding/json"
+	"os"
+
+	dapputil "github.com/privatix/dappctrl/util"
+)
+
+// Variables has a store environment variables.
+type Variables struct {
```
Comment (furkhat) [inline env/env.go]: I suggest to name it `Config`, `InstallationConfig` or similar

File: env/env.go [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+// Write saves the variables to json file.
+func (v *Variables) Write(path string) error {
+	write, err := os.Create(path)
+	if err != nil {
+		return err
+	}
+	defer write.Close()
+
+	return json.NewEncoder(write).Encode(v)
```
Comment (furkhat) [inline env/env.go]: Are you sure we should write json to `.env` file?

File: env/env.go (hunk 1 above)
Comment (furkhat) [inline env/env.go]: I suggest to name it `Config`, `InstallationConfig` or similar

File: env/env.go (hunk 2 above)
Comment (furkhat) [inline env/env.go]: Are you sure we should write json to `.env` file?

Comment (sofabeat) [reply env/env.go]: This was my suggestion. All our config files are in JSON format. Why should we use something special here? @furkhat

Comment (ubozov) [reply env/env.go]: Maybe rename to `.env.json`?

File: env/env.go (hunk 1 above)
Comment (ubozov) [reply env/env.go]: updated

**Pull request merged**